app/events/views.py

```diff
@@ -26,26 +27,19 @@ def create():
 	return jsonify({"message" : "please login or register  to create an event"}), 401
 
 @events.route('/all')
-def get_all():
+@events.route('/all/<int:page>')
+def get_all(page=1):
 	"""fetch all events available"""
 	if g.user:
-		events = Events.get_all()
-		if events:
+		#fetch the first 15 events based on event date
+		result = Events.query.order_by(Events.event_date.asc()).paginate(page=page, per_page=1, error_out=False)
```

Comment:
Why did you use the argument `.order_by`?

Reply:
so that it retrieves the events whose date is closer to the current date, assuming the deletion of events with past dates is implemented.